src/screens/NewProfile/NewProfile.js

```diff
@@ -30,12 +30,10 @@ import { BaseText, MediumText, Paragraph } from 'components/legacy/Typography';
 import Button from 'components/legacy/Button';
 import ProfileImage from 'components/ProfileImage';
 import Checkbox from 'components/legacy/Checkbox';
-import PrismicDocumentModal from 'components/Modals/PrismicDocumentModal';
 import TextInput from 'components/legacy/TextInput';
-import Modal from 'components/Modal';
 
 // Constants
-import { PERMISSIONS, SET_WALLET_PIN_CODE } from 'constants/navigationConstants';
+import { PERMISSIONS, SET_WALLET_PIN_CODE, TERMS_AND_PRIVACY } from 'constants/navigationConstants';
```

Comment:
Could we change this to `LEGAL_MODAL` or something similar for consistency with the rest of the naming please?